docs/robot_advanced.rst

```diff
@@ -117,32 +117,68 @@ With the ``PageObjects`` library, you can define classes that represent page obj
+Page objects are normal Python classes that use the ``pageobject`` decorator provided by CumulusCI. Unlike traditional Robot Framework keyword libraries, you can easily define and use keywords in multiple classes within a single file.
 
-To create a page object class, start by inheriting from one of the provided base classes. No matter which class you inherit from, your page object class gets these predefined properties.
+To create a page object class, start by inheriting from one of the provided base classes. You need to use the ``pageobject`` decorator to designate the class as a page object, and to describe the type of page (Listing, Detail, etc) and the associated salesfore object. From within a test, these page objects are referenced using both the type and object name (eg: ``Go to page  Listing  CustomObject__c``).
 
-* ``self.object_name``: The name of the object related to the class. This is defined via the ``object_name`` parameter to the ``pageobject`` decorator. Do not add the namespace prefix in the decorator. This attribute will automatically add the prefix from the ``cumulusci.yml`` file when necessary.
-* ``self.builtin``: A reference to the Robot Framework ``BuiltIn`` library that can be used to directly call built-in keywords. Any built-in keyword can be called by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.builtin.log`` and ``self.builtin.get_variable_value``).
-* ``self.cumulusci``: A reference to the CumulusCI keyword library. Call any keyword in this library by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.cumulusci.get_org_info``).
-* ``self.salesforce``: A reference to the Salesforce keyword library. Call any keyword in this library by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.salesforce.wait_until_loading_is_complete``).
-* ``self.selenium``: A reference to SeleniumLibrary. Call any keyword in this library by converting the name to all lowercase, and replacing all spaces with underscores (such as ``self.selenim.wait_until_page_contains_element``).
+The following example illustrates how to create a ``Listing`` page object for ``CustomObject__c``.
+
+.. code-block:: python
+
+    from cumulusci.robotframework.pageobjects import ListingPage, pageobject
 
+    @pageobject(page_type='Listing', object_name='CustomObject__c')
+    class CustomObjectListingPage(ListingPage):
+        ...
+
+Using object aliases
+""""""""""""""""""""
+
+Within a test, if you want to refer to the page object with a more human-readable name such as ``Custom Object`` rather than ``CustomObject__c`` you can do so by setting ``object_name`` to ``Custom Object`` and then defining ``_object_name`` in the class, like in the following example.
```

Comment:
```suggestion
Within a test, if you want to refer to the page object with a more human-readable name such as ``Custom Object`` rather than ``CustomObject__c`` you can do so by setting ``object_name`` to ``Custom Object`` and then defining ``_object_name`` in the class, as in the following example.
```